package.json

```diff
@@ -30,6 +31,7 @@
     "node": ">=4"
   },
   "dependencies": {
+    "@types/jquery": "^2.0.39",
```

Comment:
Could we use [`optionalDependencies`](https://docs.npmjs.com/files/package.json#optionaldependencies) instead?

We don't need these across all uses of jQuery Once.